lib/cusolver/dense.jl

```diff
@@ -494,6 +494,131 @@ for (bname, fname, elty, relty) in ((:cusolverDnSgesvdj_bufferSize, :cusolverDnS
     end
 end
 
+for (bname, fname, elty, relty) in ((:cusolverDnSgesvdjBatched_bufferSize, :cusolverDnSgesvdjBatched, :Float32, :Float32),
+                                    (:cusolverDnDgesvdjBatched_bufferSize, :cusolverDnDgesvdjBatched, :Float64, :Float64),
+                                    (:cusolverDnCgesvdjBatched_bufferSize, :cusolverDnCgesvdjBatched, :ComplexF32, :Float32),
+                                    (:cusolverDnZgesvdjBatched_bufferSize, :cusolverDnZgesvdjBatched, :ComplexF64, :Float64))
+    @eval begin
+        function gesvdj!(jobz::Char,
+                         econ::Int,
```

Comment:
Shouldn't this be a simple Bool?

Reply:
My mistake. I was copying the structure of the unbatched gesvdj. The batched one doesn't take an econ argument and always does full sized svd (see api [here](https://docs.nvidia.com/cuda/cusolver/#cusolverdn-t-gesvdjbatched)). It is fixed now.